src/database.ts

```diff
@@ -1989,12 +1997,27 @@ class Database extends common.GrpcServiceObject {
    * });
    * ```
    */
-  getTransaction(): Promise<[Transaction]>;
```

Comment:
Isn't removing this overload of this function a breaking change?

Reply:
This isn't a breaking change here since we have modified the signature to: 
```
getTransaction(
    optionsOrCallback?: GetTransactionOptions
  ): Promise<[Transaction]>;
```

where optionsOrCallback is an optional parameter which we may skip to pass while calling getTransaction()

Reply:
ack